Assets/Scripts/Utility/RMBLayout.cs

```diff
@@ -703,7 +702,7 @@ public static bool IsRepairShop(DFLocation.BuildingTypes buildingType)
                 {
                     // Get model transform
                     Vector3 modelPosition = new Vector3(obj.XPos, -obj.YPos, obj.ZPos) * MeshReader.GlobalScale;
-                    Vector3 modelRotation = new Vector3(0, -obj.YRotation / BlocksFile.RotationDivisor, 0);
+                    Vector3 modelRotation = new Vector3(0 -obj.YRotation / BlocksFile.RotationDivisor, 0);
```

Comment:
Is this change correct? Looks wrong to remove the comma here.

Reply:
Good find. Fixed.